packages/checkout-com/nuxt/proxyMiddleware.js

```diff
@@ -0,0 +1,79 @@
+const axios = require('axios');
+const express = require('express');
+const app = express();
+
+let ckoPublicKey = null;
+let ckoSecretKey = null;
+
+app.use(express.json());
+
+const sendJsonResponse = (res, json) => {
+  res.setHeader('Content-Type', 'application/json');
+  res.end(json);
+};
+
+const sendError = (res, errorCode, errorMessage) => res.status(errorCode).send(errorMessage);
+
+const apiRequestHeaders = (ckoSecretKey) => ({
+  headers: {
+    authorization: ckoSecretKey,
+    'Content-Type': 'application/json'
+  }
+});
+const getStoredMethods = async ({ ckoPublicKey, ckoSecretKey, customerId }) => {
+  try {
+    const { data } = await axios.get(
+      `https://play-commercetools.cko-playground.ckotech.co/merchants/${ckoPublicKey}/customers/${customerId}`,
```

Comment:
that should configurable it can't be hardcoded so move It (`https://play-commercetools.cko-playground.ckotech.co`) to the config file